src/test/fuzz/txrequest.cpp

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) 2020 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <primitives/transaction.h>
+#include <random.h>
+#include <txrequest.h>
+#include <test/fuzz/fuzz.h>
+#include <crypto/common.h>
+#include <crypto/siphash.h>
```

Comment:
8d0dd46b4fcdf4c664747e638eda046f8d51079c
nit: Lexicographic order for headers?

Reply:
Done.